src/sentry/notifications/notifications/base.py

```diff
@@ -180,14 +164,16 @@ def get_sentry_query_params(
         return f"?referrer={self.get_referrer(provider, recipient)}"
 
     def get_settings_url(self, recipient: Team | User, provider: ExternalProviders) -> str:
-        # settings url is dependant on the provider so we know which provider is sending them into Sentry
+        # Settings url is dependant on the provider so we know which provider is sending them into Sentry.
         if isinstance(recipient, Team):
             team = Team.objects.get(id=recipient.id)
-            url_str = f"/settings/{self.org_slug}/teams/{team.slug}/notifications/"
+            url_str = f"/settings/{self.organization.slug}/teams/{team.slug}/notifications/"
```

Comment:
I know this isn't really in the diff but couldn't this be replaced with `recipient.slug` and remove the DB round trip on L169?

Reply:
Yup great catch